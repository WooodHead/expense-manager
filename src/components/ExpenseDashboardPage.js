import React from 'react';
import ExpenseList from './ExpenseList';
import ExpeListFilter from './ExpenseListFilter';
import ExpensesSummary from './ExpensesSummary';

const ExpenseDashboardPage = () => ( 
    <div> 
        <ExpensesSummary />
        <ExpeListFilter />    
        <ExpenseList />
    </div> 
);

export default ExpenseDashboardPage;