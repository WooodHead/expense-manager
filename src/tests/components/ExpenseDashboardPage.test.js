// react-test-renderer
import React from 'react';
import { shallow } from 'enzyme';
// import ReactShallowRenderer from 'react-test-renderer/shallow';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';

test('should render header properly', () => {
    const wrapper = shallow(<ExpenseDashboardPage />);
    expect(wrapper).toMatchSnapshot();
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
});
