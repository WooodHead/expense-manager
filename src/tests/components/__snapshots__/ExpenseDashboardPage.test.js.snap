// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header properly 1`] = `
<div>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseListFilter) />
  <Connect(ExpenseList) />
</div>
`;
